### ISSUE TITLE
Enqueue command doesn't work for podcasts with ocnews as urls-source

### ISSUE THREAD

**Issue by doronbehar** [newsboat/newsboat#147]
### Version Info

```
$ newsboat -v
newsboat 2.10.2 - https://newsboat.org/
Copyright (C) 2006-2015 Andreas Krennmair
Copyright (C) 2015-2017 Alexander Batischev
Copyright (C) 2006-2017 Newsbeuter contributors
Copyright (C) 2017 Newsboat contributors

newsboat is free software and licensed under the MIT License.
Type `newsboat -vv' for more information.

newsboat 2.10.2
System: Linux 4.15.9-1-ARCH (x86_64)
Compiler: g++ 7.2.1 20171215
ncurses: ncurses 6.1.20180127 (compiled with 6.0)
libcurl: libcurl/7.58.0 OpenSSL/1.1.0g zlib/1.2.11 libidn2/2.0.4 libpsl/0.20.0 (+libidn2/2.0.4) nghttp2/1.30.0 (compiled with 7.57.0)
SQLite: 3.22.0 (compiled with 3.21.0)
libxml2: compiled with 2.9.7
```
Nextcloud version: `13.0.1` (stable) Sitting on an Arch Linux VPS with Apache.

### Steps to reproduce the issue

1. Use a urls-source of `ocnews` and specify a server address and username and password. 
2. Add an example podcast feed to your nextcloud instance.
3. Try to open newsboat and enqueue a podcast episode.

### Result

Newsboat is writing on it's console: `Invalid URL: ''`.

### Note

With a `"local"` `urls-source`, the episode can be queued just fine.

**Comment by Minoru** [comment]
Indeed! [The API](https://github.com/owncloudarchive/news/wiki/Items-1.2) actually provides `enclosureMime` and `enclosureLink` fields, we just don't use them.

@dirb, are you still using ownCloud News? Would you be able to add support for enclosures? Seems like it's sufficient to populate `rss_item`'s `enclosure_url` and `enclosure_type` if `utils::is_valid_podcast_type` returns `true` for the latter, but I don't have an ownCloud installation to test the result.

**Comment by doronbehar** [comment]
Here is the relevant API documentation for Nextcloud about the enclosure url: https://github.com/nextcloud/news/blob/master/docs/externalapi/External-Api.md#items

Is it different then OwnCloud's?

**Comment by Minoru** [comment]
It's a newer version, supported only by Nectcloud. We use version 1.2, which is supported by both Own- and Nextcloud.

**Comment by doronbehar** [comment]
You mean this: https://github.com/nextcloud/news/blob/master/docs/externalapi/Legacy.md#items ?

**Comment by Minoru** [comment]
Yep, that's the one. I didn't actually compare, but I believe this is the same document as the one I've linked to.

**Comment by dirb** [comment]
I did a [pull request](https://github.com/newsboat/newsboat/pull/148), I haven't used the podcast support but it populates `~/.newsboat/queue` with the urls and the names of the files to download, I don't know  if it was what it was supposed to do.

**Comment by Minoru** [comment]
That's the correct behaviour, yes. Thank you!

**Comment by Minoru** [comment]
Fixed by 7c1c4ff9ec3e43a83b4d36f96e9b55b9692430c3.
